restart_process/Tiltfile

```diff
@@ -13,18 +12,16 @@ KWARGS_BLACKLIST = [
 
 
 def docker_build_with_restart(ref, context, entrypoint, live_update,
-                              base_suffix='-base', restart_file=RESTART_FILE, **kwargs):
+                              base_suffix='-base', shell=False, **kwargs):
```

Comment:
need add `shell` to the arg comments below

is there value in adding this parameter vs telling them to pass entrypoint as array in string? it feels inconsistent with our other APIs

Reply:
yeahhh i go back and forth. it IS inconsistent with our other APIs, but the other APIs aren't exactly intuitive and we expect this extension to have a much higher surface area than those APIs we're trying to be consistent with.

you're probably right tho :-/